Project-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import './Display.css';
 
-const CalculatorDisplay = props => {
-  return <div className="display">0</div>
+const CalculatorDisplay = ({ displayValue }) => {
+  return <div className="display">{displayValue}</div>
 };
 
```

Comment:
- Noble great job tackling the calculator Project. You managed to add events listeners for displayed  numbers👍
- Great way using Switch case in JS.
- Your code is clear and i like how You are using .map() generate new components and pass props to them. This is how You create reusable components.
- Splending git workflow.
Overall great job 👍